tests/lib/prepare.sh

```diff
@@ -164,6 +164,11 @@ EOF
     fi
 
     if [[ "$SPREAD_SYSTEM" == debian-* || "$SPREAD_SYSTEM" == ubuntu-* ]]; then
+        if [[ "$SPREAD_SYSTEM" == ubuntu-* ]]; then
```

Comment:
actually pollinate is "just" a single-file bash script; we could ship it as part of the build to places where it doesn't already exist. Something for another PR though.